### DISCUSSION TITLE
Is there a way to make the MODS menu persist when loading a new mod?

### DISCUSSION THREAD

**Discussion by vide0hanz** [Novum/vkQuake#669] [Q&A] [answered]
It'd be nice to load up a mod and be able to switch to a new one without having to exit the current mod first. Is this possible or would this cause compatibility issues with some mods which have custom menus?

**Comment by Novum** [comment, marked as answer]
It's because there is no font for the big letters at all. It's just whole images for "Single Player" etc. So I can't just make it write "Mods", I had to manually create an image and it only fits the unmodded Quake style. E.g. it would look completely out of place with Alkaline's menu.

Having to click Quit and then Mods is really not that bad.

**Comment by vide0hanz** [comment]
Makes sense. I suspected the menu entries were images when I poked around the menu code a few days ago. This answers my question, thanks! 

I agree its not that bad to have to quit a mod before changing to a new one, and if my question came off as a complaint I certainly did not intend for it to. Thank you for all of your work on maintaining this source port :)
